examples/requirements/src/language-server/requirements-lang-validator.ts

```diff
@@ -4,30 +4,22 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-import { ValidationAcceptor, ValidationChecks, ValidationRegistry } from 'langium';
+import { ValidationAcceptor, ValidationChecks } from 'langium';
 import { RequirementsAndTestsAstType, Requirement, isTestModel } from './generated/ast';
-import { RequirementsLangServices } from './requirements-lang-module';
+import type { RequirementsLangServices } from './requirements-lang-module';
```

Comment:
Why does this `import type` thing show up here? We don't use it anywhere else, do we?

Reply:
We use it to ensure that there are not runtime dependencies to that file. In this case a runtime dependency would cause a cycle and thus unexpected problems like undefined global definitions.